test/integration/cartridge_role_test.lua

```diff
@@ -668,3 +670,81 @@ g.test_invalig_global_labels_names = function()
         }
     }, 'label name is not allowed to be "zone" or "alias"')
 end
+
+g.test_include_metrics = function()
+    local server = g.cluster.main_server
+    server:upload_config({
+        metrics = {
+            export = {
+                {
+                    path = '/metrics',
+                    format = 'json'
+                },
+            },
+            include = {
+                'vinyl', 'luajit',
+            }
+        }
+    })
+
+    local resp = server:http_request('get', '/metrics', {raise = false})
+    t.assert_equals(resp.status, 200)
+
+    local metrics_cnt = #resp.json
+    local vinyl_metrics = fun.iter(resp.json):filter(function(x)
+        return x.metric_name:find('tnt_vinyl')
+    end):length()
+    local lj_metrics = fun.iter(resp.json):filter(function(x)
+        return x.metric_name:find('lj_')
+    end):length()
+    local cartridge_metrics = fun.iter(resp.json):filter(function(x)
```

Comment:
I don't get it. `cartridge_issues` and `clock` metrics are disabled, isn't they?

Reply:
It's from previous version. Cartridge metrics count is 0 now. I'll delete it